packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -140,17 +140,21 @@ Function Get-Java
       }
     }
 
-    if ($javaVersion -eq '') { Write-Verbose 'Unable to determine Java version' }
     if ($javaPath -eq '') { Write-Error "Unable to determine the path to java.exe"; return $null }
     if ($javaCMD -eq '') { $javaCMD = "$javaPath\bin\java.exe" }
     if (-not (Test-Path -Path $javaCMD)) { Write-Error "Could not find java at $javaCMD"; return $null }
- 
-    $ShellArgs = @()
 
     Write-Verbose "Java detected at '$javaCMD'"
-    Write-Verbose "Java version detected as '$javaVersion'"
+
+    $javaVerInfo = Get-JavaVersion -Path $javaCMD
+    if ($javaVerInfo.Version -eq '') {
```

Comment:
I don't know about idiomatic.  I return a hashtable with empty string values instead of `null`.  Alternatively I could cast it to a string, which is basically the same thing